src/main.ts

```diff
@@ -8,7 +8,7 @@ import { modules } from './modules/modules';
 const { discord } = config;
 
 const client = new Client({
-  intents: getIntentsFromModules(modules),
+  intents: ['Guilds', ...getIntentsFromModules(modules)],
```

Comment:
why do we need that ?

Reply:
for this : https://github.com/codinglab-io/discord-bot/pull/72/files#diff-cabbe10dd99e98f945b0d166d48dd9e5acf554a9ceb22d5d052da39f11d13995R21